Assets/Scripts/UI/PickupPup/DogProfile/DogProfile.cs

```diff
@@ -69,18 +65,16 @@ public virtual void SetProfile(Dog dog)
 
         nameText.text = dogInfo.Name;
         breedText.text = dogInfo.Breed.Breed;
-        for(int i = 0; i < descriptionText.Length; i++)
-        {
-            if (i < dogInfo.Descriptions.Length)
-            {
-                descriptionFields[i].Show();
-                descriptionText[i].text = dogInfo.Descriptions[i];
-            }
-            else
-            {
-                descriptionFields[i].Hide();
-            }
-        }
+		if (descriptionText != null) {
```

Comment:
Paren spacing and bracket newline note for this enter block (also applies to `else` statements)